test/blackbox-tests/test-cases/configurator-import.t/my_stub.c

```diff
@@ -0,0 +1,12 @@
+/* File curses_stubs.c -- stub code for curses */
+#include <curses.h>
+#include <caml/mlvalues.h>
+#include <caml/memory.h>
+#include <caml/alloc.h>
+#include <caml/custom.h>
```

Comment:
Is it possible to reduce this test not to depend on headers from OCaml and curses? It's best if the tests are as isolated as possible.